public/js/selfoss-shares.js

```diff
@@ -8,11 +8,8 @@ selfoss.shares = {
         this.enabledShares = enabledShares;
         this.initialized = true;
 
-        this.register('delicious', 'd', function(url, title) {
-            window.open('https://delicious.com/save?url=' + encodeURIComponent(url) + '&title=' + encodeURIComponent(title));
-        });
-        this.register('googleplus', 'g', function(url) {
-            window.open('https://plus.google.com/share?url=' + encodeURIComponent(url));
+        this.register('diaspora', 'd', function(url, title) {
+            window.open('https://share.diasporafoundation.org/?url='+encodeURIComponent(url)+'&title='+encodeURIComponent(title)+'&notes='+'');
```

Comment:
`+` needs to be spaced from both sides.

Reply:
Also why is `'&notes='+''` needed?

Reply:
" + " Oh, really? Works in chromium... but no prob, will fix. And remove the &notes as well.

Reply:
Btw. very nice piece of software! Never seen one where such a change requires only touching one file.

Reply:
Yeah, it is just a style preference – we are trying to have an uniform style everywhere and have a scripts to enforce that. If you have the development environment set up, you can use `npm run lint:client -- --fix` command to convert the code to the preferred style automatically.

Reply:
Travis was complaining about the "+" vs. " + ". That's fixed with another commit.
Graphics is from here:
https://www.deviantart.com/creativekaizen/art/Diaspora-Icon-Set-257241499
Vector is in .ai-file (Adobe Illustrator, i guess). Inkscape can open & save as .svg. Should i upload that somewhere?

Reply:
file:///src/net/selfoss-2.18/diaspora_icons/diaspora_icons_cs5.svg

Reply:
The upload here is crap. Doesn't even like a 10-Byte-.txt-file :(

Reply:
A link should be enough.

Reply:
Ok, there it is:
http://www.69b.org/diaspora.svg